control_plane/src/endpoint.rs

```diff
@@ -824,11 +824,12 @@ impl Endpoint {
         // cleanup work to do after postgres stops, like syncing safekeepers,
         // etc.
         //
-        // If destroying, send it SIGTERM before waiting. Sometimes we do *not*
-        // want this cleanup: tests intentionally do stop when majority of
-        // safekeepers is down, so sync-safekeepers would hang otherwise. This
-        // could be a separate flag though.
-        self.wait_for_compute_ctl_to_exit(destroy)?;
+        // If destroying or stop mode is immediate, send it SIGTERM before
+        // waiting. Sometimes we do *not* want this cleanup: tests intentionally
```

Comment:
Some tests are using "immediate" shutdown mode to test abnormal compute termination.
I wonder if this change makes this tests less useful.

Reply:
I'd say it should only make them more useful. That is, immediate means as fast as possible, killing compute_ctl is even more 'immediate'.